Game.obj : error LNK2019: SDL_GetError"public: bool __cdecl Game::Initialize(void)" (?Initialize@Game@@QEAA_NXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
Game.obj : error LNK2019: SDL_CreateWindow"public: bool __cdecl Game::Initialize(void)" (?Initialize@Game@@QEAA_NXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
Game.obj : error LNK2019: SDL_DestroyWindow"public: void __cdecl Game::Shutdown(void)" (?Shutdown@Game@@QEAAXXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
Game.obj : error LNK2019: SDL_Log"public: bool __cdecl Game::Initialize(void)" (?Initialize@Game@@QEAA_NXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
Game.obj : error LNK2019: SDL_Init"public: bool __cdecl Game::Initialize(void)" (?Initialize@Game@@QEAA_NXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
Game.obj : error LNK2019: SDL_Quit"public: void __cdecl Game::Shutdown(void)" (?Shutdown@Game@@QEAAXXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
Game.obj : error LNK2019: "private: void __cdecl Game::ProcessInput(void)" (?ProcessInput@Game@@AEAAXXZ)"public: void __cdecl Game::RunLoop(void)" (?RunLoop@Game@@QEAAXXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
Game.obj : error LNK2019: "private: void __cdecl Game::UpdateGame(void)" (?UpdateGame@Game@@AEAAXXZ)"public: void __cdecl Game::RunLoop(void)" (?RunLoop@Game@@QEAAXXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
Game.obj : error LNK2019: "private: void __cdecl Game::GenerateOutput(void)" (?GenerateOutput@Game@@AEAAXXZ)"public: void __cdecl Game::RunLoop(void)" (?RunLoop@Game@@QEAAXXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
Main.obj : error LNK2019: "public: __cdecl Game::Game(void)" (??0Game@@QEAA@XZ)SDL_main 함수에서 참조되는 확인할 수 없는 외부 기호
MSVCRTD.lib(exe_main.obj) : error LNK2019: main"int __cdecl invoke_main(void)" (?invoke_main@@YAHXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
E:\code\External\SDL\lib\win\x86\SDL2.lib : warning LNK4272: 'x86' 라이브러리 컴퓨터 종류가 'x64' 대상 컴퓨터 종류와 충돌합니다.
E:\code\External\SDL\lib\win\x86\SDL2main.lib : warning LNK4272: 'x86' 라이브러리 컴퓨터 종류가 'x64' 대상 컴퓨터 종류와 충돌합니다.
E:\SDL\Chapter1\x64\Debug\Game.exe : fatal error LNK1120: 11개의 확인할 수 없는 외부 참조입니다.
